presto-docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -262,6 +262,25 @@ as part of the table name, separated by a colon. For example:
 .. _full text query: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#query-string-syntax
 
 
+Pass-through Queries
+--------------------
+
+The Elasticsearch connector allows you to embed any valid Elasticsearch query,
+that uses the `ES Query DLS
+<https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl.html>`_
+in your SQL query.
+
+The results can then be used in any SQL statement, wrapping the ES query. The
+syntax extends the syntax of the enhanced ES table names with the following::
+
+    SELECT * FROM es.default."<index>$query:<es-query>"
+
+The ES query string ``es-query`` is base32-encoded to avoid having to deal with
+escaping quotes and case sensitivity issues in table identifiers.
+
+The result of these query tables is a table with a single row and a single
+column named ``result`` of type JSON.
```

Comment:
The result is VARCHAR, not JSON, but it contains the JSON payload returned by Elasticsearch, and can be processed with the built-in JSON functions in presto (https://prestosql.io/docs/current/functions/json.html)

Reply:
Updated